docs/metrics/index.md

```diff
@@ -150,3 +104,61 @@ biases, and its intended uses.
     and [recall](recall.md).
 
 </div>
+
+## :kolena-metrics-nlp-icon-20: Natural Language Processing
+
+<div class="grid cards" markdown>
+
+- [BERTScore](bertscore.md)
+
+    ---
+
+    BERTScore is a metric used in NLP workflows to measure textual similarity between candidate texts and reference
+    texts.
+
+- [BLEU](bleu.md)
+
+    ---
+
+    BLEU is a metric commonly used in a variety of NLP workflows to evaluate the quality of candidate texts. BLEU can be
+    thought of as an analog to [precision](precision.md) for text comparisons.
+
+- [METEOR](meteor.md)
+
+    ---
+
+    METEOR is a widely recognized and vital metric used in NLP. It is used to measure the quality of candidate texts
+    against reference texts. Though it is an n-gram based metric, it goes beyond traditional methods by factoring in
+    elements such as precision, recall, and order to provide a comprehensive measure of text quality.
+
+- [ROUGE-N](rouge-n.md)
+
+    ---
+
+    ROUGE-N, a metric within the broader ROUGE metric collection, is a vital metric in the field of NLP. It assesses
+    the quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts.
+    ROUGE-N can be thought of as an analog to [recall](recall.md) for text comparisons.
+
+</div>
+
+## :kolena-metrics-cv-icon-20: Computer Vision
```

Comment:
Minor nit — I'd arrange computer vision above NLP both on this page and in the sidebar.